clang/lib/Sema/AnalysisBasedWarnings.cpp

```diff
@@ -2265,7 +2265,7 @@ class UnsafeBufferUsageReporter : public UnsafeBufferUsageHandler {
       } else if (isa<MemberExpr>(Operation)) {
         // note_unsafe_buffer_operation doesn't have this mode yet.
         assert(!IsRelatedToDecl && "Not implemented yet!");
-        auto ME = dyn_cast<MemberExpr>(Operation);
+        auto ME = cast<MemberExpr>(Operation);
```

Comment:
NIT: afaik, coding guidelines suggest adding `*` 
```suggestion
        auto *ME = cast<MemberExpr>(Operation);
```

Reply:
Thank you @Fznamznon for catching and reviews. Fixed now.